src/BizHawk.Client.Common/movie/bk2/Bk2Movie.ModeApi.cs

```diff
@@ -26,14 +26,14 @@ public virtual bool Stop(bool saveChanges = true)
 			bool saved = false;
 			if (saveChanges)
 			{
-				if (Mode == MovieMode.Record || (this.IsActive() && Changes))
+				// How would a movie ever have changes while inactive?

```

Comment:
Adding this comment while simultaneously making this exact situation possible is probably not useful.

Reply:
You mean the check in `MovieSession`? That's true, so the active check should probably not exist here in the first place... to be honest I don't even know whether you can even get a movie into the `MovieSession` in an inactive state with the current code.